### DISCUSSION TITLE
Renovate has stopped running with Mend Renovate hosted app

### DISCUSSION THREAD

**Discussion by lordcodes** [renovatebot/renovate#20529] [Request Help]
### I'm using the Mend Renovate hosted app.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

It was working great until November 2022, however, it hasn't ran since then. It is running for different repository in the same organisation perfectly fine.

If I log into the Renovate app dashboard there are no jobs or logs shown for the repository it isn't working on. I have tried checking the checkbox on the Dependency Dashboard, but nothing happened after a long time.

The repository is 'github/getbusy-plc/workiro-ios'.

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
It looks like this repo may be timing out. I'm attempting to run it in the app in dryRun mode to see how far it can get if it's not creating PRs or doing lookups yet.

**Reply by rarkins** [reply]
Even with dryRun=extract it appears to be timing out. How big is this repo when you clone it?

**Reply by lordcodes** [reply]
Thank you for looking into it!

On GitHub settings it says the repository is 290MB. I did have a problem with large files historically in the repository, would renovate be affected by this, or does it just check out the latest commit without any history?

**Reply by rarkins** [reply]
It does a blobless clone, then checks out the default branch. So the history is there, but it should skip any old deleted blobs

**Reply by lordcodes** [reply]
Okay thanks. There were some frameworks and large files in history that not only contributed large blobs but also many refs and made a regular Git clone much larger than necessary. I seem to have slimmed the repository down a lot.

Will see if renovate will re-run now and not time out.
